.babelrc

```diff
@@ -83,7 +83,7 @@
         "transform-runtime"
       ],
       "presets": [
-        "env",
+        ["env", { "modules": false }],
```

Comment:
I'd recommend to refactor this whole file. I usually just create `.babelrc.js` for better control (you have to put `{"presets": ["./.babelrc.js"]}` in you `.babelrc` so babel6 can handle it) and then I can just use javascript to construct plugins/presets based on `process.env` variables. It's imho hard to see what changes between various variants here, because of overwhelming duplication

Reply:
Oh, I'd also recommend using `loose` mode with `modules: false` in call cases + smth like
```js
const cjs = process.env.NODE_ENV === 'test' || process.env.BABEL_ENV === 'cjs'
// ...
plugins: [
 // ...
 cjs && 'transform-es2015-modules-commonjs'
].filter(Boolean)
```

Why? `loose` mode for everything except commonjs transform, because it makes `__esModule` property enumerable, so I consider default mode better for this case.